--- xbmc-13.1-Gotham/tools/Linux/xbmc.sh.in.0214~	2014-06-04 07:16:52.000000000 +0200
+++ xbmc-13.1-Gotham/tools/Linux/xbmc.sh.in	2014-06-07 07:41:38.709101085 +0200
@@ -25,20 +25,6 @@ datarootdir="@datarootdir@"
 LIBDIR="@libdir@"
 CRASHLOG_DIR=${CRASHLOG_DIR:-$HOME}
 
-# Check for some options used by this script
-while [ "$#" -gt "0" ]
-do
-    case "$1" in
-        --setlibdir)
-            LIBDIR="$2"
-            shift; shift
-            ;;
-        *)
-            shift
-            ;;
-    esac
-done
-
 single_stacktrace()
 {
   # core filename is either "core.$PID" or "core"
@@ -114,7 +100,7 @@ print_crash_report()
   echo "Crash report available at $FILE"
 }
 
-python @datadir@/xbmc/FEH.py $SAVED_ARGS
+python @datadir@/xbmc/FEH.py "$@"
 RET=$?
 if [ $RET -ne 0 ]; then
   exit $RET
@@ -132,7 +118,7 @@ LOOP=1
 while [ $(( $LOOP )) = "1" ]
 do
   LOOP=0
-  "$LIBDIR/xbmc/xbmc.bin" $SAVED_ARGS
+  "$LIBDIR/xbmc/xbmc.bin" "$@"
   RET=$?
   if [ $(( $RET == 65 )) = "1" ]
   then # User requested to restart app
