diff -urN xbmc-13.0-Gotham-patched/xbmc/network/upnp/UPnPInternal.cpp xbmc-13.0-Gotham/xbmc/network/upnp/UPnPInternal.cpp
--- xbmc-13.0-Gotham/xbmc/network/upnp/UPnPInternal.cpp	2014-05-05 03:25:01.691809419 +1100
+++ xbmc-13.0-Gotham-patched/xbmc/network/upnp/UPnPInternal.cpp	2014-05-05 03:28:11.725808018 +1100
@@ -364,7 +364,7 @@
         rooturi = NPT_HttpUrl("localhost", upnp_server->GetPort(), "/");
     }
 
-    if (!item.m_bIsFolder) {
+    if (!item.m_bIsFolder && !item.IsPlayList() && !item.IsSmartPlayList()) {
         object = new PLT_MediaItem();
         object->m_ObjectID = item.GetPath();
 
