diff -urN xbmc-13.0-Gotham/xbmc/cores/dvdplayer/DVDInputStreams/DllDvdNav.h xbmc-13.0-Gotham-patched/xbmc/cores/dvdplayer/DVDInputStreams/DllDvdNav.h
--- xbmc-13.0-Gotham/xbmc/cores/dvdplayer/DVDInputStreams/DllDvdNav.h	2014-05-05 05:24:54.192690312 +1100
+++ xbmc-13.0-Gotham-patched/xbmc/cores/dvdplayer/DVDInputStreams/DllDvdNav.h	2014-05-05 05:25:30.179261539 +1100
@@ -26,10 +26,6 @@
 
  #include "dvdnav/dvdnav.h"
 
- #ifndef WIN32
- #define WIN32
- #endif // WIN32
-
  #ifndef HAVE_CONFIG_H
  #define HAVE_CONFIG_H
  #endif
