diff -urN xbmc-11.0-orig/lib/DllAvFilter.h xbmc-11.0-patched/lib/DllAvFilter.h
--- xbmc-11.0-orig/lib/DllAvFilter.h	2012-11-20 03:39:32.466767549 +1100
+++ xbmc-11.0-patched/lib/DllAvFilter.h	2012-11-20 03:44:04.000000000 +1100
@@ -132,10 +132,10 @@
   {
     return ::avfilter_graph_config(graphctx, log_ctx);
   }
-  virtual int avfilter_poll_frame(AVFilterLink *link) { return ::avfilter_poll_frame(link); }
-  virtual int avfilter_request_frame(AVFilterLink *link) { return ::avfilter_request_frame(link); }
-  virtual int av_vsrc_buffer_add_frame(AVFilterContext *buffer_filter, AVFrame *frame, int flags) { return ::av_vsrc_buffer_add_frame(buffer_filter, frame, flags); }
-  virtual AVFilterBufferRef *avfilter_get_video_buffer(AVFilterLink *link, int perms, int w, int h) { return ::avfilter_get_video_buffer(link, perms, w, h); }
+  virtual int avfilter_poll_frame(AVFilterLink *link) { return avfilter_poll_frame(link); }
+  virtual int avfilter_request_frame(AVFilterLink *link) { return avfilter_request_frame(link); }
+  virtual int av_vsrc_buffer_add_frame(AVFilterContext *buffer_filter, AVFrame *frame, int flags) { return av_vsrc_buffer_add_frame(buffer_filter, frame, flags); }
+  virtual AVFilterBufferRef *avfilter_get_video_buffer(AVFilterLink *link, int perms, int w, int h) { return avfilter_get_video_buffer(link, perms, w, h); }
   virtual void avfilter_unref_buffer(AVFilterBufferRef *ref) { ::avfilter_unref_buffer(ref); }
   virtual int avfilter_link(AVFilterContext *src, unsigned srcpad, AVFilterContext *dst, unsigned dstpad) { return ::avfilter_link(src, srcpad, dst, dstpad); }
   virtual int av_buffersink_get_buffer_ref(AVFilterContext *buffer_sink, AVFilterBufferRef **bufref, int flags) { return ::av_buffersink_get_buffer_ref(buffer_sink, bufref, flags); }
