diff -urN xbmc-12.1/xbmc/network/upnp/UPnPInternal.cpp xbmc-12.1-patched/xbmc/network/upnp/UPnPInternal.cpp
--- xbmc-12.1/xbmc/network/upnp/UPnPInternal.cpp	2013-03-21 03:14:42.000000000 +1100
+++ xbmc-12.1-patched/xbmc/network/upnp/UPnPInternal.cpp	2013-03-30 14:29:38.668105769 +1100
@@ -238,7 +238,7 @@
         if (tag.m_type == "musicvideo") {
           object.m_ObjectClass.type = "object.item.videoItem.musicVideoClip";
           object.m_Creator = StringUtils::Join(tag.m_artist, g_advancedSettings.m_videoItemSeparator);
-          object.m_Title = tag.m_strTitle;
+          object.m_Title = NPT_String::Format("%s - %s", (char *)object.m_Creator, tag.m_strTitle.c_str());
           object.m_ReferenceID = NPT_String::Format("videodb://3/2/%i", tag.m_iDbId);
         } else if (tag.m_type == "movie") {
           object.m_ObjectClass.type = "object.item.videoItem.movie";
