diff -urN xbmc-13.0-Gotham/xbmc/network/upnp/UPnPInternal.cpp xbmc-13.0-Gotham-patched/xbmc/network/upnp/UPnPInternal.cpp
--- xbmc-13.0-Gotham/xbmc/network/upnp/UPnPInternal.cpp	2014-05-04 08:03:25.000000000 +1100
+++ xbmc-13.0-Gotham-patched/xbmc/network/upnp/UPnPInternal.cpp	2014-05-05 03:25:01.691809419 +1100
@@ -272,7 +272,7 @@
         if (tag.m_type == "musicvideo") {
           object.m_ObjectClass.type = "object.item.videoItem.musicVideoClip";
           object.m_Creator = StringUtils::Join(tag.m_artist, g_advancedSettings.m_videoItemSeparator);
-          object.m_Title = tag.m_strTitle;
+          object.m_Title = NPT_String::Format("%s - %s", (char *)object.m_Creator, tag.m_strTitle.c_str());
           object.m_ReferenceID = NPT_String::Format("videodb://musicvideos/titles/%i", tag.m_iDbId);
         } else if (tag.m_type == "movie") {
           object.m_ObjectClass.type = "object.item.videoItem.movie";
