diff -urN xbmc-12.2/xbmc/network/upnp/UPnPInternal.cpp xbmc-12.2-patched/xbmc/network/upnp/UPnPInternal.cpp
--- xbmc-12.2/xbmc/network/upnp/UPnPInternal.cpp	2013-05-03 16:57:41.000000000 +1100
+++ xbmc-12.2-patched/xbmc/network/upnp/UPnPInternal.cpp	2013-05-14 21:56:44.117736411 +1100
@@ -328,7 +328,7 @@
         ips.Insert(ips.GetFirstItem(), context->GetLocalAddress().GetIpAddress());
     }
 
-    if (!item.m_bIsFolder) {
+    if (!item.m_bIsFolder && !item.IsPlayList() && !item.IsSmartPlayList()) {
         object = new PLT_MediaItem();
         object->m_ObjectID = item.GetPath();
 
